recipes/flowtorch/LICENSE.txt

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
The license is already included in the source distribution on PyPI so you can remove the file. During built, `conda-built` will just get it from the downloaded package.

Reply:
Wasn't sure if it would find the LICENSE.txt file (it was in a another folder inside the archive from PyPI). Had added it as a fail-safe. 

Thank you, for pointing it out. Will remove it in next commit.

Reply:
You can specify the path to the license like you'd normally would with any directory `path/to/license`. If conda-build cannot find the license it will fail the build so you won't accidentally create a package with no license.

Reply:
Done.